### DISCUSSION TITLE
[Dev Tools] Wrong files in nightly release

### DISCUSSION THREAD

**Discussion by thepluck** [zkSync-Community-Hub/zksync-developers#741] [Dev Tooling] [answered]
### Environment

Mainnet

### Select the Dev Tool you are using

Foundry

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

![image](https://github.com/user-attachments/assets/e1c1ad0c-0cda-49e9-8cd8-83348f2ac3cf)
As in the image, the file [foundry_nightly_darwin_amd64.tar.gz](https://github.com/matter-labs/foundry-zksync/releases/download/nightly/foundry_nightly_darwin_amd64.tar.gz) is showed instead of foundry_nightly_linux_amd64.tar.gz.

**Comment by alhenaai** [comment]
It seems like there might be an issue with the file naming or availability in the nightly release of Foundry for ZKsync. I recommend checking the latest releases on the GitHub repository to ensure you are accessing the correct files. If the issue persists, please raise this concern in the [ZKsync developers discussions on GitHub](https://github.com/zkSync-Community-Hub/zksync-developers/discussions) where the development team can provide more direct assistance and possibly rectify any errors in the release files.

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by uF4No** [comment]
The release contains both arm64 and amd64 builds for Linux: https://github.com/matter-labs/foundry-zksync/releases/tag/nightly-baee07edbcafac2ac011c5e918a9672d192b8fad

**Reply by thepluck** [reply]
I mean the nightly release without a specified version [link](https://github.com/matter-labs/foundry-zksync/releases/tag/nightly). The [toolchain](https://github.com/bgd-labs/foundry-zksync-toolchain) that I use download binaries from it.

**Reply by MexicanAce** [reply, marked as answer]
@thepluck Apologies for the delayed response, but this was fixed on Oct 23rd around 2 hours after your reply. We had to make some fixes after we pulled changes from upstream `foundry`. Thanks again for reporting 🙏 !
